compiler/damlc/tests/daml-test-files/InterfaceSyntax.daml

```diff
@@ -0,0 +1,41 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- @SINCE-LF-FEATURE DAML_INTERFACE
+-- @ENABLE-SCENARIOS
+
+module InterfaceSyntax where
+
+interface I where
+  m0 : Bool -> Party
+  m1 : Either Int Bool -> Text
+
+-- empty interfaces with 'where'
+interface J where
+interface K where
```

Comment:
Is it possible to remove this `where`, for empty interfaces? I think it would be nice for consistency sake, but not super important. (But happy with leaving that for later.)

Reply:
yes! I'll do that in a separate set of PRs since the GHC one is already kinda big on its own